addons/mission/functions/fnc_moduleAmbientLogistics.sqf

```diff
@@ -0,0 +1,71 @@
+#include "script_component.hpp"
+/*
+    Author:
+        Bridg
+
+    Description:
+        Creates a logistics run every x amount of seconds
+        Will stop when all the 'runs' have been used up
+
+    Parameters:
+        0: The module logic <OBJECT>
+
+    Return value:
+        Nothing
+*/
+(_this select 1) params ["_logic"];
+
+private _destinations = synchronizedObjects _logic;
+private _time = _logic getVariable [QGVAR(ambientLogisticsInterval), 0];
+private _units = parseSimpleArray (_logic getVariable [QGVAR(ambientLogisticsUnitPoolString), []]);
+private _vehicles = parseSimpleArray (_logic getVariable [QGVAR(ambientLogisticsVehiclePoolString), []]);
+private _side = _logic getVariable [QGVAR(ambientLogisticsSide), 0];
+private _numberOfRunsRemaining = _logic getVariable [QGVAR(ambientLogisticsNumberOfRuns), 0];
+
+[{
+    params ["_args", "_idPFH"];
+    _args params ["_destinations", "_units", "_vehicles", "_side", "_logic", "_numberOfRunsRemaining"];
+
+    if (_numberOfRunsRemaining <= 0) exitWith {
+        [_idPFH] call cba_fnc_removePerFrameHandler;
+    };
+
+    _args set [5, _numberOfRunsRemaining - 1];
+
+    _side = switch (_side) do {
+        case 0: { east };
+        case 1: { west };
+        case 2: { resistance };
+    };
+
+    // create vehicle and crew
+    private _logiSpawnPos = getPos _logic;
```

Comment:
Would prefer to keep vehicle and unit spawning in one place, use `common,spawnGroupVehicle`. Can't see anything here that it doesn't support